cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3740,7 +3723,7 @@ grade=signed
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, fmt.Sprintf("%s-model-measured", s.sysLabel)), testutil.FilePresent)
 }
 
-func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencryptedDataSaveEncryptedUnhappy(c *C) {
+func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedEncryptedDataUnencryptedSaveUnhappy(c *C) {
```

Comment:
The old scenario is not really possible in this new world. What we had before was unencrypted data, which then proceeds to unlock encrypted save and that only works because of the fallback behavior of secboot.UnlockVolumeUsingSealedKeyIfencrypted(). In the new world, once we know that data is unencrypted, we only try the unencrypted save path, which expliclitly passes an empty key and disables the recovery key prompt. I guess I could have left the old test and verify that we indeed as not passing the keys and that options are correct.

Reply:
Sorry my previous comment was confusing. In the new world, once we know that data is unencrypted, we never try to use secboot to open ubuntu save, instead the code goes to find a partition with ubuntu-save label and mount it.

Reply:
>   I guess I could have left the old test and verify that we indeed as not passing the keys and that options are correct.

yes, in this kind of situation if the old test high-level setup at least if not behavior/checks can be kept around with some changes/reduction, it's probably best